hazelcast/src/test/java/com/hazelcast/spi/impl/operationexecutor/slowoperationdetector/SlowOperationDetectorBasicTest.java

```diff
@@ -46,6 +46,21 @@ public void teardown() {
     }
 
     @Test
+    public void testDisabled() {
+        Config config = new Config();
+        config.setProperty(GroupProperties.PROP_SLOW_OPERATION_DETECTOR_ENABLED, "false");
+        config.setProperty(GroupProperties.PROP_SLOW_OPERATION_DETECTOR_THRESHOLD_MILLIS, "1000");
+
+        instance = getSingleNodeCluster(config);
+
+        getInternalOperationService(instance).execute(new SlowRunnable(3, Operation.GENERIC_PARTITION_ID));
```

Comment:
Make use of the HazelcastTestSupport.getOperationService method which returns the InternalOperationService.